### DISCUSSION TITLE
Need documentation around reasons to upgrade Capacitor, if capacitor is needed for latest OS's

### DISCUSSION THREAD

**Discussion by inorganik** [ionic-team/capacitor#6746] [General]
Obviously we need to keep software up-to-date for many reasons. But when you have a number of community plugins that may not be supported in the latest version of capacitor, you have to weigh the tradeoffs. 

I would like to suggest the capacitor team give us the main reasons and benefits for each major update, so we can determine the priority of doing upgrade work in our apps, and which breaking changes we may not be able to support. And also if we need the latest version to support the latest iOS/Android. Thanks!

**Comment by robingenz** [comment]
Here you can find the Capacitor Support Policy which should answer some of your questions: https://capacitorjs.com/docs/main/reference/support-policy

**Reply by inorganik** [reply]
Thanks! That does help

**Comment by jcesarmobile** [comment]
Google requires to target the latest SDK every year by August, Capacitor major versions are created to properly target the latest SDK before the google deadline.

While you could not update the capacitor version and only change the targetSDK version in your app, that always cause issues on some plugins when running in Android 13 devices if they need new code to properly work there.
